wmfdata/spark.py

```diff
@@ -129,20 +129,27 @@ def create_custom_session(
             category=FutureWarning
         )
 
+    old_session = get_active_session()
+    if old_session:
+        old_session.stop()
```

Comment:
I notice you switched from stopping the context to stopping the session. Just curious: did you do this just because they're basically the same thing and this lets you reuse `get_active session`? Or is there some relevant difference here?

Reply:
> did you do this just because they're basically the same thing and this lets you reuse get_active session?

This.

Reply:
Okay, makes sense. I like the new approach!